### PR TITLE
Add tests on GitHub Actions

### PR DESCRIPTION
**Pull request by mziyut**
## What

- Add tests on GitHub Actions
    - Added v17 and v18 to the executable Node versions
    - Eliminate test execution on Travis CI
    - Support for Ruby v3 was not implemented.
- Upgrade bundler version

## Why

- Node version is out of date and has not worked in the past.

## Reference

- https://github.com/increments/camo/pull/5

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - run: bundle install
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: "npm"
+      - run: npm install
+      - run: npm run build
+      - run: npm run start &
```
Comment (kyntk) [inline .github/workflows/test.yml:36]: Maybe this is unnecessary
```suggestion
      - run: npm run start
```

Comment (kyntk) [inline .github/workflows/test.yml:36]: Maybe this is unnecessary
```suggestion
      - run: npm run start
```

**Review by kyntk — APPROVED** [review]
LGTM!

**Review by mayobimu — APPROVED** [review]
✅GTM!

**Pull request merged**